torch/csrc/jit/tensorexpr/expr.cpp

```diff
@@ -430,13 +430,13 @@ BufHandle Buf::make(const std::vector<ExprHandle>& dims, Dtype dtype) {
 }
 
 BufHandle Buf::make(
-      const std::string& name_hint,
-      const std::vector<ExprHandle>& dims,
-      Dtype dtype,
-      c10::optional<ExprHandle> initializer,
-      c10::optional<std::vector<ExprHandle>> strides,
-      c10::optional<ExprHandle> qscale,
-      c10::optional<ExprHandle> qzero) {
+    const std::string& name_hint,
+    const std::vector<ExprHandle>& dims,
+    Dtype dtype,
+    c10::optional<ExprHandle> initializer,
+    c10::optional<std::vector<ExprHandle>> strides,
+    c10::optional<ExprHandle> qscale,
+    c10::optional<ExprHandle> qzero) {
```

Comment:
This was added in the previous PR, right? Wondering why clang-format didn't catch it there..

Reply:
Hm, yeah, maybe the previous PR breaks clang-format :)